src/test/groovy/com/gradle/TestDevelocityInjection.groovy

```diff
@@ -109,6 +109,32 @@ class TestDevelocityInjection extends BaseInitScriptTest {
             .findAll {testGradle, testDvPlugin -> testDvPlugin.compatibleCCUDVersion != null}
     }
 
+    def "applies CCUD plugin via init script with DV plugin already applied using develocity.buildCache"() {
+        given:
+        declareDvPluginApplication(gradleVersion, testDvPlugin)
+        settingsFile << """
+buildCache {
+    remote(${buildCacheExtension}.buildCache)
+}
+"""
+        when:
+        def config = testConfig().withCCUDPlugin(CCUD_PLUGIN_VERSION)
+        def result = run(GRADLE_8_X, config)
```

Comment:
Shouldn't it be? (I realize it's functionally equivalent)

```suggestion
        def result = run(gradleVersion, config)
```

Reply:
👍🏼 fixed